### DISCUSSION TITLE
Stuck in text box after leaving insert mode

### DISCUSSION THREAD

**Discussion by fritzrehde** [qutebrowser/qutebrowser#7044] [Q&A] [answered]
Once I leave insert mode by pressing Escape, the input field is still selected (and e.g. j and k don't work to scroll down and up on the whole page). How do I exit / "unfocus" this focused text box (repeatadly pressing Escape doesn't work)?
Thank you in advance!

**Comment by toofar** [comment]
You need to apply a workaround locally currently, see https://github.com/qutebrowser/qutebrowser/issues/2668#issuecomment-589795263 for an example of re-binding Escape to both leave mode and unfocus any element.
(Also go +1 that issue if it helps you!)

**Comment by fritzrehde** [comment, marked as answer]
Thank you! In case someone else is also looking for a solution to the problem, I am now using the following binding:
```
config.bind('<Escape>', 'mode-leave ;; jseval -q document.activeElement.blur()', mode='insert')
```
